atomix/cluster/src/main/java/io/atomix/raft/partition/impl/RaftPartitionServer.java

```diff
@@ -257,6 +258,23 @@ public void addCommittedEntryListener(final RaftCommittedEntryListener commitLis
     server.getContext().addCommittedEntryListener(commitListener);
   }
 
+  /**
+   * @see
+   *     io.atomix.raft.impl.RaftContext#addSnapshotReplicationListener(SnapshotReplicationListener)
+   */
+  public void addSnapshotReplicationListener(final SnapshotReplicationListener logResetListener) {
+    server.getContext().addSnapshotReplicationListener(logResetListener);
+  }
+
+  /**
+   * @see
+   *     io.atomix.raft.impl.RaftContext#removeSnapshotReplicationListener(SnapshotReplicationListener)
+   */
+  public void removeSnapshotReplicationListener(
+      final SnapshotReplicationListener logResetListener) {
+    server.getContext().removeSnapshotReplicationListener(logResetListener);
```

Comment:
Fixing the parameter name.

```suggestion
  public void removeSnapshotReplicationListener(
      final SnapshotReplicationListener listener) {
    server.getContext().removeSnapshotReplicationListener(listener);
```